content/blog/eng/php-day-2020.md

```diff
@@ -0,0 +1,78 @@
+---
+authors: ["veronica-bolognesi"]
+comments: true
+date: "2020-09-10"
+draft: false
+share: true
+categories: [English, Agile, Scrum, Kanban]
+title: "PHPDay 2020"
+
+languageCode: "en-EN"
+type: "post"
+---
+On sept 8th the [2020 edition of PHPDay](https://2020.phpday.it/) was held in a remote fashion.
+
+Having experienced the 2019 edition (indeed my first one, despite being a PHP developer from eons!) I should say that it was a little difficult to maintain the concentration for such a long event, but the effort was well worth of, given the level of both lecturers and talks.
```

Comment:
I would change the sentence as follows:

(...) I would say that it was a bit diffcult to stay focused for such a long event. However, given the high-quality level of lectures and talks, it was totally worth the effort.

Reply:
@Zolletta: ok, are you keeping the original sentence?